feature-libs/quote/components/details/comment/quote-details-comment.component.html

```diff
@@ -1,20 +1,25 @@
 <ng-container *ngIf="quoteDetails$ | async as quoteDetails">
-  <div class="quote-comment-toggle" (click)="showComments = !showComments">
-    <cx-icon
-      aria-hidden="false"
-      [type]="showComments ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
+  <ng-container *ngIf="showComments(quoteDetails)">
+    <div
+      class="quote-comment-toggle"
+      (click)="expandComments = !expandComments"
     >
-    </cx-icon>
-    <span class="quote-comment-text">{{
-      'quote.comments.title' | cxTranslate
-    }}</span>
-  </div>
-  <div *ngIf="showComments">
-    <cx-messaging
-      [messageEvents$]="messageEvents$"
-      [messagingConfigs]="messagingConfigs"
-      (send)="onSend($event, quoteDetails.code)"
-      (itemClicked)="onItemClicked($event)"
-    ></cx-messaging>
-  </div>
+      <cx-icon
+        aria-hidden="false"
+        [type]="expandComments ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
```

Comment:
Please, define a method for this query to make the code more readable.

Reply:
i think this code is very readable and wrapping this into a method just makes is look more complex than it is

Reply:
@Uli-Tiger, the review is for putting out a suggestion. It is up to you to follow it or not.
I would understand this part of the code better if I see something like `toggleIcon` instead of this condition